src/accelerate/hooks.py

```diff
@@ -179,6 +181,10 @@ def remove_hook_from_module(module: nn.Module):
         module.forward = module._old_forward
         delattr(module, "_old_forward")
 
+    if recurse and len(list(module.children())) > 1:
+        for _, child in module.named_children():
```

Comment:
```suggestion
        for child in module.children():
```
No need to ask for the names if you don't want them ;-)